server/asset/dcr/dcr.go

```diff
@@ -251,6 +254,26 @@ func (dcr *Backend) ValidateSecret(secret, contract []byte) bool {
 	return bytes.Equal(h[:], secretHash)
 }
 
+// Synced is true if the blockchain is ready for action.
+func (dcr *Backend) Synced() (bool, error) {
+	chainInfo, err := dcr.node.GetBlockChainInfo()
+	if err != nil {
+		return false, fmt.Errorf("GetBlockChainInfo error: %w", err)
+	}
+	if chainInfo.Headers-chainInfo.Blocks > 1 {
+		return false, nil
+	}
+	if dcr.net == dex.Mainnet {
+		_, err = dcr.FeeRate()
+		if err != nil {
+			dcr.log.Debugf("Synced = false because of FeeRate error = %v", err)
+		}
```

Comment:
I actually thought that this was suggested in #501. I'm removing the `false` return value, but we should consider other solutions for a default fee on the server. Especially with bitcoin max fees now >= 300 sats/vbyte, during periods of low activity, max fees might be 5x or 10x the fees we'd get from `estimatesmartfee` on a primed node.

Reply:
Not sure about 5x or 10x even, but a certain percent over the `estimatesmartfee` result seems like a great idea given how easy it is to get buried by other txns where blocks are taking too long.  I was thinking more like 150% though, so it would indicate 450 for 300 e.g. and that would have been more than sufficient throughout this latest congestion.